nautobot/cloud/tables.py

```diff
@@ -44,7 +44,7 @@ class Meta(BaseTable.Meta):
 class CloudNetworkTable(BaseTable):
     pk = ToggleColumn()
     name = tables.Column(linkify=True)
-    cloud_type = tables.Column(linkify=True)
+    cloud_resource_type = tables.Column(linkify=True)
     cloud_account = tables.Column(linkify=True)
     parent = tables.Column(linkify=True)
     actions = ButtonsColumn(CloudNetwork)
```

Comment:
Maybe need to add here too:

```suggestion
    tags = TagColumn(url_name="cloud:cloudnetwork_list")
    actions = ButtonsColumn(CloudNetwork)
```